docs/source/troubleshooting/lvrt-crash-dumps.rst

```diff
@@ -0,0 +1,199 @@
+request a memory or core dump from the system at the time of the
+crash. These crash dumps may provide further insight into the type of
+crash or where the crash is occurring, as well as information about the
+system. However, it's important to keep in mind that crash dumps are not
+guaranteed to solve an issue or even provide useful insight. They are
+another troubleshooting tool which can be useful in narrowing down a
+problem or understanding things further. 
+
+| The more context given with a crash dump, the more likely it is to
+  have useful information. This can include:
+
+-  Logs of what the application is doing at the time of the
+   crash
+-  Simplified reproducing cases
+-  Context as to what changed when the crash started happening
+
+| Without this information, the likelihood that a dump file will provide
+  useful information decreases significantly. For example, a core dump
+  may show an access violation due to a bad refrence in the code but it
+  may not show what caused the reference to become bad in the first
+  place. That information can be used to narrow things down but will not
+  give a root cause on its own.
+
+| Additionally, reproducing a crash internally to NI is generally
+  preferred. A dump can be unwound externally, but without access to NI
+  symbols, it is unlikely to yield useful information. Additionally, if 
+  the issue is reproduced internally at NI, the same system can be used 
+  for both reproducing and unwinding the core dump ensuring that the 
+  software versions used for debugging the dump file are consistent.
+  
+
+Generating a Core Dump
+========================
+
+Configuring a Target for Core Dumps
+-----------------------------------
+
+| By default, NI Linux Real-Time targets have core dumps disabled. This
+  setting is to ensure that these files - which can be quite large - do
+  no take up valuable storage space on embedded devices when not
+  needed. To enable core dumps, follow the steps below.
+
+#. Ensure that the target has software installed, is in run-mode, and
+   will not reboot into safe mode upon reboot. This can be done by
+   verifying the following in NI MAX:
+
+   -  Verify that the **Status** of the target in
+      the **System Settings** tab is currently **Connected - Running.**
+   -  Verify that the **Force Safe Mode** checkbox is **not** checked in
+      the **System Settings** tab.
+
+#. On the **System Settings** tab in NI MAX, check the boxes
+   under **Startup Settings** for both **Enable Secure Shell Server
+   (sshd)** and **Enable Console Out.**
+
+#. Enable core dumps via one of the following methods:
+
+   #. Run the following command via a terminal (serial or SSH) on the target: 
+
+      .. code:: bash
+
+         sed -e 's|^[#[:space:]]*ulimit [[:space:]]*-c [[:space:]]*.*$|ulimit -c unlimited|' -i /etc/init.d/lvrt-wrapper
+
+   
+   #. Modify the */etc/init.d/lvrt-wrapper* file in a text editor via FTP,
```

Comment:
NIT: Can we change these paths and code-segments from italics to literals?

Reply:
Done.